src/include/common/system_message.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <cerrno>
+#include <string>
+#include <system_error>
+
+namespace kuzu {
+namespace common {
+inline std::string system_err_message(int code) {
```

Comment:
Is it intentional to use snake case for function name?

Reply:
No, good catch.